src/main/java/net/snowflake/ingest/streaming/internal/ColumnProperties.java

```diff
@@ -0,0 +1,59 @@
+package net.snowflake.ingest.streaming.internal;
+
+/**
```

Comment:
Could you add a comment to explain the relationship with `ColumnMetdata`?

Reply:
Absolutely